### PR TITLE
removed deprecated api's details from release-3.6.0

### PR DESCRIPTION
**Pull request by Hari-stackroute**

### CODE REVIEW

**Comment by keshavprasadms** [comment]
Hello @Hari-stackroute,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://693.qa.docs.sunbird.org/3.6.0/ .<br/> **Note:** Only _3.6.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://693.qa.docs.sunbird.org/3.6.0/apis/badgingframeworkapi// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/badgingframeworkapi// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/bulkupload// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/firebasecloudmessagingapi// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/firebasecloudmessagingapi// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/geolocationapi// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/geolocationapi// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/masterkeyapi// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/masterkeyapi// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/objectapi// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/objectapi// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/userapi// <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @Hari-stackroute,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://693.qa.docs.sunbird.org/3.6.0/ .<br/> **Note:** Only _3.6.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://693.qa.docs.sunbird.org/3.6.0/apis/badgingframeworkapi// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/badgingframeworkapi// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/bulkupload// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/firebasecloudmessagingapi// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/firebasecloudmessagingapi// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/geolocationapi// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/geolocationapi// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/masterkeyapi// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/masterkeyapi// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/objectapi// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/objectapi// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/userapi// <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @Hari-stackroute,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://693.qa.docs.sunbird.org/3.6.0/ .<br/> **Note:** Only _3.6.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://693.qa.docs.sunbird.org/3.6.0/apis/badgingframeworkapi// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/badgingframeworkapi// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/bulkupload// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/firebasecloudmessagingapi// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/firebasecloudmessagingapi// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/geolocationapi// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/geolocationapi// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/masterkeyapi// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/masterkeyapi// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/objectapi// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/objectapi// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/userapi// <br />  <br/> **Modified sections are -** <br />

File: apis/userapi/userapi.yaml [hunk 1]
```diff
@@ -1624,45 +1004,44 @@ paths:
...
         AllowDynamicQueryParameters: false
         AllowDynamicFormParameters: false
         IsMultiContentStreaming: false
-  /user/v1/skill/read:
+  /user/v1/unblock:
     post:
       description: >-
-        This API is used to read user added skill.
-           - The endpoint for **Get user skill** is `/user/v1/skill/read`
+        This API is used to unban/unblock a user
```
Comment (Sunitashelar) [inline apis/userapi/userapi.yaml:1016]: @Hari-stackroute The word here should be 'remove the ban' or 'release' because unban is not a word.

**Review by Sunitashelar — CHANGES_REQUESTED** [review]

**Comment by keshavprasadms** [comment]
Hello @Hari-stackroute,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://693.qa.docs.sunbird.org/3.6.0/ .<br/> **Note:** Only _3.6.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://693.qa.docs.sunbird.org/3.6.0/apis/badgingframeworkapi// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/badgingframeworkapi// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/bulkupload// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/firebasecloudmessagingapi// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/firebasecloudmessagingapi// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/geolocationapi// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/geolocationapi// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/masterkeyapi// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/masterkeyapi// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/objectapi// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/objectapi// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/userapi// <br />  <br/> **Modified sections are -** <br />

File: apis/userapi/userapi.yaml (hunk 1 above)
Comment (Hari-stackroute) [reply apis/userapi/userapi.yaml]: @Sunitashelar : done

**Comment by keshavprasadms** [comment]
Hello @Hari-stackroute,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://693.qa.docs.sunbird.org/3.6.0/ .<br/> **Note:** Only _3.6.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://693.qa.docs.sunbird.org/3.6.0/apis/badgingframeworkapi// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/badgingframeworkapi// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/bulkupload// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/firebasecloudmessagingapi// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/firebasecloudmessagingapi// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/geolocationapi// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/geolocationapi// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/masterkeyapi// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/masterkeyapi// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/objectapi// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/objectapi// <br /> - http://693.qa.docs.sunbird.org/3.6.0/apis/userapi// <br />  <br/> **Modified sections are -** <br />

**Review by Sunitashelar — APPROVED** [review]

**Pull request merged**